docker-compose-airgapped.yml

```diff
@@ -0,0 +1,462 @@
+---
```

Comment:
I am hesitant to keep a separate composition - can the original and AG compositions be a single file, with any changes being passed via environment?

Reply:
I'm not sure how to do that. Have you got any pointers?

What I can certainly see is turning the `docker-compose-airgapped.yml` into an "overlay" that just modifies the original `docker-compose.yml`

Reply:
FYI I'm working on using the docker compose `profiles` to keep it a single file.